src/esm_runscripts/prepcompute.py

```diff
@@ -40,9 +41,9 @@ def compile_model(config):
     if not version:
         return config
     if config.get("general", {}).get("run_number") == 1:
-        print("First year, checking if we need to compile...")
+        logger.info("First year, checking if we need to compile...")
         if not config.get("general", {}).get("use_compiled_model", True):
-            print(f"Huhu --> compiling {model}-{version}")
+            logger.debug(f"Huhu --> compiling {model}-{version}")
```

Comment:
```suggestion
            logger.info(f"Huhu --> compiling {model}-{version}")
```
Shouldn't this be `info`?

Reply:
Replaced debug by info